documentation/book/proc-customizing-deployments.adoc

```diff
@@ -0,0 +1,56 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='proc-customizing-deployments-{context}']
+= Customizing deployments
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `template` property in the `Kafka`, `KafkaConnect`, `KafkaConnectS2I`, or `KafkaMirrorMaker` resource.
```

Comment:
comma before `KafkaMirrorMaker`

Reply:
@stanlyDoge ??? It is already there.

Reply:
Oh, I thought it should not be there.

Reply:
I think it should ... they are always adding it there when you have before it more than one thing. Let's see if @laidan6000 likes it.